library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -646,7 +645,8 @@ public Cursor query(Query query) {
         if (underlyingCursor == null) {
             return null;
         }
-        return new CursorTranslator(underlyingCursor, downloadsUriProvider.getDownloadsByBatchUri());
+        StatusTranslator statusTranslator = new StatusTranslator();
```

Comment:
:thought_balloon:  thought that but then each query message new's up the query translator so this is a dependency of that newing up, not of the downloadmanager. I think it should stay here? :star2: